Network/doc/NetworkPlugin.md

```diff
@@ -958,6 +1009,14 @@ Network interface events:
 ## *onConnectionStatusChanged <sup>event</sup>*
 
 Triggered when a connection is made or lost.
+ 
+### Methods
+  
+| Method | Description | 
+| :----------- | :----------- |
+| `setDefaultInterface` | Sets the default interface successfully and triggers this event |
```

Comment:
The "setDefaultInterface" method may trigger this event (not guaranteed).
For example: If the device's default interface is already WIFI and "setDefaultInterface(WIFI)" is invoked (successfully), there will be no change to the device's default interface and thus no onConnectionStatusChanged or onDefaultInterfaceChanged events generated.

Reply:
Updated as per suggestion.